sdk/kusto/Azure.ResourceManager.Kusto/tests/Prerequisites/Scripts/CreateTestResources.ps1

```diff
@@ -1,8 +1,10 @@
 ﻿Import-Module Az.Kusto
 
-$SubscriptionId = "e8257c73-24c5-4791-94dc-8b7901c90dbf" # Kusto_Dev_Kusto_Ilay_04_Test
+#$SubscriptionId = "e8257c73-24c5-4791-94dc-8b7901c90dbf" # Kusto_Dev_Kusto_Ilay_04_Test
```

Comment:
remove the commented scripts if they are not in user anymore.

Reply:
removed